packages/hooks/src/useInvoiceCreate.ts

```diff
@@ -94,14 +104,17 @@ export const useInvoiceCreate = ({
     'endDate',
   ]);
 
-  const { data: tokens } = useFetchTokens();
+  const { data: tokens } = useFetchTokens({ enabled: !!networkConfig });
```

Comment:
This looks like it will only be run if `networkConfig` is not null, but this `networkConfig` is not being used in the SI app. Again this will break the SI app. Please test this